index.html

```diff
@@ -208,9 +208,46 @@ <h3 class="cardHeader aboutmeHeader">Skills</h3>
 
     </section>
 
-    
+    <section class="contactme centeredGrid">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form action="https://formspree.io/f/mleaonkj" method="POST" id="contactmeForm">
+            <input class="userinput" type="text" name="name" id="userName" maxlength="30" placeholder="Full name" required>
+            <input class="userinput" type="email" name="email" id="userEmail" placeholder="Email address" required>
+            <textarea class="userinput" name="message" id="userMessage" cols="30" rows="10" maxlength="500" placeholder="Enter your message" required></textarea>
+        </form>
+        <button class="button" type="submit" form="contactmeForm">Get In touch</button>
```

Comment:
- [ ] It would be better if we implement the required design template, this is an essential skill. Being able to implement a Figma design is a great addition to our skill. Let us compare our design with the Figma design. ( Paying close attention to the border color)
`Project`
![Screenshot from 2021-08-17 19-29-06](https://user-images.githubusercontent.com/46388113/129788867-02e7e823-05ea-4fba-bfcc-64d67d66f574.png)

`Figma`
![Screenshot from 2021-08-17 19-28-38](https://user-images.githubusercontent.com/46388113/129789032-7e63829c-4654-4f00-9179-8e9501e29332.png)